build.rs

```diff
@@ -7,4 +7,16 @@ fn main() {
             err
         );
     }
+
+    use std::env;
+    if let Ok(target) = env::var("TARGET") {
+        match target.as_ref() {
+            "asmjs-unknown-emscripten" => { },
+            "wasm32-unknown-emscripten" => { },
+            "wasm32-unknown-unknown" => { },
+            _ => println!("cargo:rustc-cfg=parallel"),
+        }
+    } else {
+        panic!("Could not identify the building target (missing TARGET env variable, are you sure you are running this from cargo?)");
```

Comment:
If someone was running it without cargo, it shouldn't panic either.

Reply:
This panic was meant for someone trying to run build.rs outside of cargo, which has no reason to happen except if it's a mistake from the user.

Reply:
That's fine like this then ;)